### DISCUSSION TITLE
Contract problem

### DISCUSSION THREAD

**Discussion by kaiw0866** [zkSync-Community-Hub/zksync-developers#414] [Contract Verification] [answered]
### Environment

Testnet

### Issue Type

- [X] Plugin
- [X] Block Explorer

### Contract Address

0x1013fe3fd2c295c3a0d4000bc391dfb9513e611e

### Compiler Type

Single file

### zkSolc Version

0

### Solc Version

0

### Contract Name

kaiwang

### Contract Code

![Snipaste_2024-03-14_15-58-31](https://github.com/zkSync-Community-Hub/zksync-developers/assets/163388770/56d58e59-ac50-4f95-a89a-ef1fd16e9362)

### Constructor Arguments

0

**Comment by kaiw0866** [comment]
I have a problem. I have submitted two transactions to the ETH main network respectively, and the first transaction cost is relatively low, resulting in withdrawal failure. Can I combine the two transactions? That way you don't have to pay as much GAS

**Reply by dutterbutter** [reply, marked as answer]
Batch transactions are not supported in the UI at this time.
